src/style.css

```diff
@@ -1 +1,117 @@
-/* add styles here */
+:root {
+  --main-text-color: #3d4e61;
+  --white-color: #fff;
+  --light-white-color: #f6f6f7;
+  --main-border-color: #e1e7ed;
+  --primary-font: 16px;
+  --small-font: 14px;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf) format('ttf');
```

Comment:
The format specified in the `src` attribute should be 'truetype' instead of 'ttf'. The correct format is `format('truetype')`.